data/api/server-server/keys_query.yaml

```diff
@@ -35,99 +26,131 @@ paths:
       parameters:
         - in: path
           name: serverName
-          type: string
           description: The server's DNS name to query
           required: true
-          x-example: matrix.org
+          example: matrix.org
+          schema:
+            type: string
         - in: path
           name: keyId
-          type: string
           description: |-
             **Deprecated**. Servers should not use this parameter and instead
             opt to return all keys, not just the requested one. The key ID to
             look up.
 
             When excluded, the trailing slash on this endpoint is optional.
           required: false
```

Comment:
This is invalid in OpenAPI 3.x, all path parameters must have `required:true`.

Should we define two separate paths and deprecate the one with the `keyId`?

Reply:
To be honest we could just drop `keyId` from the spec. It's been deprecated for years, and nobody should be relying on it.